<ui version="4.0" >
 <class>modCalcGalCoordDlg</class>
 <widget class="QWidget" name="modCalcGalCoordDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>487</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>546</width>
        <height>431</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>8</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Equatorial Coordinates (J2000)</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>8</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="QLabel" name="decLabel" >
              <property name="text" >
               <string>Declination:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="raLabel" >
              <property name="text" >
               <string>Right ascension:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="dmsBox" name="RA" />
            </item>
            <item row="1" column="1" >
             <widget class="dmsBox" name="Dec" />
            </item>
            <item row="2" column="1" >
             <widget class="QPushButton" name="ObjectButton" >
              <property name="text" >
               <string>Select Object...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3alactic" >
           <property name="title" >
            <string>Galactic Coordinates</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>8</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="1" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="dmsBox" name="GalLongitude" />
            </item>
            <item row="1" column="1" >
             <widget class="dmsBox" name="GalLatitude" />
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>507</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="2" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="epochCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
            </property>
            <property name="text" >
             <string>Epoch:</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="epochBoxBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>1950.0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QCheckBox" name="galLongCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
            </property>
            <property name="text" >
             <string>Gal. long.:</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QCheckBox" name="galLatCheckBatch" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="toolTip" >
             <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
            </property>
            <property name="text" >
             <string>Gal. lat.:</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="decCheckBatch" >
            <property name="toolTip" >
             <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
            </property>
            <property name="text" >
             <string>Dec:</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="raCheckBatch" >
            <property name="toolTip" >
             <string>If checked, value will be read from input file. If not, value will be read from adjacent box</string>
            </property>
            <property name="text" >
             <string>RA:</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="dmsBox" name="raBoxBatch" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="dmsBox" name="decBoxBatch" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="dmsBox" name="galLongBoxBatch" />
          </item>
          <item row="1" column="3" >
           <widget class="dmsBox" name="galLatBoxBatch" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Select Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>8</number>
          </property>
          <item>
           <widget class="QRadioButton" name="allRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="OutputFileBoxBatch" />
          </item>
          <item row="0" column="1" >
           <widget class="KUrlRequester" name="InputFileBoxBatch" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Input file:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>151</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="runButtonBatch" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>511</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>RA</tabstop>
  <tabstop>Dec</tabstop>
  <tabstop>ObjectButton</tabstop>
  <tabstop>GalLongitude</tabstop>
  <tabstop>GalLatitude</tabstop>
  <tabstop>raCheckBatch</tabstop>
  <tabstop>raBoxBatch</tabstop>
  <tabstop>decCheckBatch</tabstop>
  <tabstop>decBoxBatch</tabstop>
  <tabstop>epochCheckBatch</tabstop>
  <tabstop>epochBoxBatch</tabstop>
  <tabstop>galLongCheckBatch</tabstop>
  <tabstop>galLongBoxBatch</tabstop>
  <tabstop>galLatCheckBatch</tabstop>
  <tabstop>galLatBoxBatch</tabstop>
  <tabstop>allRadioBatch</tabstop>
  <tabstop>inputRadioBatch</tabstop>
  <tabstop>InputFileBoxBatch</tabstop>
  <tabstop>OutputFileBoxBatch</tabstop>
  <tabstop>runButtonBatch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
